va/validation-authority.go

```diff
@@ -117,9 +117,14 @@ func (va ValidationAuthorityImpl) validateSimpleHTTP(identifier core.AcmeIdentif
 		// connection immediately.
 		DisableKeepAlives: true,
 	}
+	logRedirect := func(req *http.Request, via []*http.Request) error {
+		va.log.Notice(fmt.Sprintf("validateSimpleHTTP [%s] redirect from %q to %q", identifier, via[len(via)-1].URL.String(), req.URL.String()))
```

Comment:
Info sounds good. I would actually like to add data like this to the authorization object, which makes it easier to find later. Filed #487 and #488.